WordPressKitTests/SiteVerticalsRequestEncodingTests.swift

```diff
@@ -0,0 +1,98 @@
+import XCTest
+@testable import WordPressKit
+
+final class SiteVerticalsRequestEncodingTests: XCTestCase {
+
+    func testSiteVerticalsRequestEncoding_WithAllParameters_IsSuccessful() {
+        // Given
+        let expectedSearch = "Landscap"
+        let expectedLimit = 5
+
+        let request = SiteVerticalsRequest(search: expectedSearch, limit: expectedLimit)
+
+        // When
+        let encoder = JSONEncoder()
+
+        XCTAssertNoThrow(try encoder.encode(request))
+        let encodedJSON = try! encoder.encode(request)
+
+        XCTAssertNoThrow(try JSONSerialization.jsonObject(with: encodedJSON, options: []))
+        let serializedJSON = try! JSONSerialization.jsonObject(with: encodedJSON, options: [])
+
+        if let _ = serializedJSON as? [String : AnyObject] {} else {
```

Comment:
Can we do 
```suggestion
        guard let jsonDictionary = serializedJSON as [String : AnyObject] else {
            XCTFail("Failed to encode a proper JSON dictionary!")
        }
```
and remove 
```suggestion
       let jsonDictionary = serializedJSON as! [String : AnyObject]
```
?

Reply:
👍 Do you think we should do the same in line 53?